src/basics.ml

```diff
@@ -112,6 +128,15 @@ let is_symb : sym -> term -> bool = fun s t ->
   | Symb(r,_) -> r == s
   | _         -> false
 
+(** [sym_cmp s s'] compares symbols [s] and [s'].  This function is designed
```

Comment:
I think you missed this comment because the previous discussion was already resolved:
>I think that the comment should be a bit more modest, because this comparison function remains quite inefficient. ;) Ideally, we would like to have unike integers associated with symbols, or at least a kind of hash. But let's not worry too much about that now. You could maybe add a FIXME explaining that this could be improved. By the way, is efficiency critical here? Is this function only used when precomputing decision trees or is it also used during interpretation?